app/assets/stylesheets/forever_style_guide/modules/_ambassador_banner.scss

```diff
@@ -86,50 +84,70 @@ a.ambassador_banner-no_ambassador {
     display: flex;
     align-items: center;
     justify-content: center;
+    flex-wrap: wrap; 
   }
 }
 
-.ambassador_banner-ambassador_dropdown-avatar {
+
+
+.ambassador_banner-dropdown-avatar {
   width: $ambassador-avatar-size;
   height: $ambassador-avatar-size;
   align-self: flex-start;
   margin: 0;
   margin-right: $padding-large-horizontal * 2;
   padding: 0;
   border-radius: 50%;
-  border: 4px solid #fafafa;
+  border: 4px solid $color-gray-100;
   box-shadow: 0 5px 20px rgba(0,0,0,.2);
 }
 
-.ambassador_banner-ambassador_dropdown-info-main-name {
+.ambassador_banner-dropdown-info {
+  max-width: 100%;
+  overflow: hidden;
+}
+
+.ambassador_banner-dropdown-info-main-name {
   margin: 0;
+  text-overflow: ellipsis;
+  overflow: hidden;
 }
 
-.ambassador_banner-ambassador_dropdown-info-secondary {
+.ambassador_banner-dropdown-info-secondary {
   margin-top: $padding-large-vertical;
 }
 
-.ambassador_banner-ambassador_dropdown-info-main-rank {
+.ambassador_banner-dropdown-info-main-rank {
   color: transparentize($color-white, 0.3);
 }
 
-.ambassador_banner-ambassador_dropdown-info-main-location {
+.ambassador_banner-dropdown-info-main-location {
   color: transparentize($color-white, 0.2);
 }
 
-a.ambassador_banner-ambassador_dropdown-info-main-manage {
-  display: block;
-  color: $color-white;
-  margin-top: $padding-default-vertical;
-}
-
-a.ambassador_banner-ambassador_dropdown-info-secondary-link {
+.ambassador_banner-dropdown-info .ambassador_banner-dropdown-info-link { // specificity needed to override style guide link styles
   display: inline-block;
-  margin-top: $padding-xs-vertical / 2;
-  margin-bottom: $padding-xs-vertical / 2;
+  max-width: 100%;
   color: $color-white;
+  text-overflow: ellipsis;
```

Comment:
I think you can @extend the existing ellipsization class or just use it on the element. Unless that's only in the store>

Reply:
Do you mean this? (Which I didn't even know we had.) It seems a little bit excessive for what we need here? 
```
@mixin multi_line_ellipsis($lineHeight: 1.1em, $lineCount: 3, $bgColor: $color-gray-700) {
  overflow: hidden;
  position: relative;
  line-height: $lineHeight;
  max-height: $lineHeight * $lineCount;
  padding-right: 1em;

  &:before {
    content: '...';
    position: absolute;
    right: 0;
    bottom: 0;
  }
  &:after {
    content: '';
    position: absolute;
    right: 0;
    width: 1em;
    height: 1em;
    margin-top: 0.2em;
    background: $bgColor;
  }
}
```

We also have this is the store, but that's all I'm aware of.
```
.break_long_string {
  overflow-wrap: break-word;
  word-wrap: break-word;
  hyphens: auto;
}
```

Reply:
I was thinking of this actually https://github.com/forever-inc/mars-ui/blob/master/src/modules/core/assets/styles/base.scss#L19

wherein there is a comment to move all of those styles to styleguide

Reply:
![wow](https://user-images.githubusercontent.com/1926679/35443276-86230ede-0278-11e8-94ad-047df29cb5b6.gif)